resources/cdk/redshift_item_tracker/README.md

```diff
@@ -0,0 +1,157 @@
+the following:
+
+```
+Outputs:
+RedshiftItemTrackerStack.ClusterId = doc-example-work-items-cluster
+RedshiftItemTrackerStack.Database = workitemtracker
+RedshiftItemTrackerStack.SecretArn = arn:aws:secretsmanager:us-west-2:0123456789012:secret:docexampleredshiftworkitems-QlmFqEXAMPLE-nsoAF0
+```
+
+### Deploying with the AWS CLI 
+
+To deploy with the AWS CLI you must first install the 
+[AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+1. Deploy AWS resources by running the following at a command prompt in the 
+    `resources/cdk/redshift_item_tracker` folder:
+    
+    ```
+    aws cloudformation create-stack --template-body file://redshift_item_tracker.yaml --stack-name YOUR_STACK_NAME
+    ```
+    
+    *Note:* The stack name must be unique within an AWS Region and AWS account. You can 
+    specify up to 128 characters, and numbers and hyphens are allowed.
+
+2. The stack takes a few minutes to deploy. You can check status by running the following:
+
+    ```
+    aws cloudformation describe-stacks --stack-name YOUR_STACK_NAME
+    ```
+    
+    When the stack is ready, it shows `StackStatus` of `CREATE_COMPLETE`.
+
+3. You can get the outputs from the stack by running the following:
+
+    ```
+    aws cloudformation describe-stacks --stack-name STACK_NAME --query Stacks[0].Outputs --output text
+    ```
+    
+    This results in output like the following: 
+    
+    ```
+    ClusterId = doc-example-work-items-cluster
+    Database = workitemtracker
+    SecretArn = arn:aws:secretsmanager:us-west-2:0123456789012:secret:docexampleredshiftworkitems-QlmFqEXAMPLE-nsoAF0
+    ```
+
+## Using the resources
+
+After the database and secret are created, you can use the AWS Management Console,
+AWS CLI, or an AWS SDK to run SQL commands to create tables and manage data in the 
+Redshift database.
+
+For example, use the AWS CLI to run a statement with the Redshift Data API Service to 
+create a table in the Redshift database. 
+Replace the `secret-arn` value with the value that was output when you deployed the 
+resource stack.
+
+```
+aws redshift-data execute-statement ^
+    --cluster-identifier "doc-example-work-items-cluster" ^
+    --database "workitemtracker" ^
+    --secret-arn "arn:aws:secretsmanager:us-west-2:0123456789012:secret:docexampleredshiftworkitems-QlmFqEXAMPLE-nsoAF0" ^
+    --sql "create table work_items (work_item_id INT GENERATED BY DEFAULT AS IDENTITY ( 1, 1 ) PRIMARY KEY, created_date TIMESTAMP DEFAULT CURRENT_TIMESTAMP, description TEXT, guide TEXT, status TEXT, username VARCHAR(45), archive BOOL DEFAULT 0);"
+```
+
+You can insert a row into the `work_items` table by substituting SQL like the following:
+
+```
+    --sql "insert into work_items (description, guide, status, username) values ('My first work item', 'My guide', 'Not started', 'myuser');"
+```
+
+Or get all data from the `work_items` table:
```

Comment:
Or --> Alternatively,

Reply:
Fixed. I shouldn't have use "or", so now it just read "You can get all..."